vncviewer/Viewport.cxx

```diff
@@ -115,9 +115,10 @@ enum { ID_DISCONNECT, ID_FULLSCREEN, ID_MINIMIZE, ID_RESIZE,
 static const WORD SCAN_FAKE = 0xaa;
 #endif
 
+
 Viewport::Viewport(int w, int h, const rfb::PixelFormat& /*serverPF*/, CConn* cc_)
   : Fl_Widget(0, 0, w, h), cc(cc_), frameBuffer(nullptr),
-    lastPointerPos(0, 0), lastButtonMask(0),
+    lastPointerPos(0, 0), mouseButtonMask(0), lastButtonMask(0),
```

Comment:
Please split the mask tracking to a separate commit. It's complex enough that it's best if we get a clear diff of what changes it entails.